packages/core/lib/modules/wallet.js

```diff
@@ -93,6 +113,8 @@ export class Wallet {
     await DockRpc.init({
       address: networkInfo.substrateUrl,
     });
+
+    this.connectionInProgress = false;
```

Comment:
this should probably be in a try/finally block so we don't get stuck thinking it's in progress even when it fails

Reply:
Fixed